go.mod

```diff
@@ -17,12 +17,12 @@ require (
 	github.com/lightninglabs/lightning-node-connect v0.1.12-alpha
 	github.com/lightninglabs/lightning-terminal/autopilotserverrpc v0.0.1
 	github.com/lightninglabs/lndclient v0.16.0-10
-	github.com/lightninglabs/loop v0.22.0-beta
+	github.com/lightninglabs/loop v0.22.1-beta
```

Comment:
Sorry, I forgot that we also have a new version of Faraday (v0.2.11-alpha). Not super important but nice to have (just new REST mappings).